File: (1) Python/(1) Python Programming Basics - September 2022/(5) Conditional Statements Advanced - Lab/fruit_shop.py
product = input()
day = input()
number_of_products = float(input())

if day == "Monday" or day == "Tuesday" or day == "Wednesday" or day == "Thursday" or day == "Friday":
    if product == "banana":
        print(f"{(2.5 * number_of_products):.2f}")
    elif product == "apple":
        print(f"{(1.2 * number_of_products):.2f}")
    elif product == "orange":
        print(f"{(0.85 * number_of_products):.2f}")
    elif product == "grapefruit":
        print(f"{(1.45 * number_of_products):.2f}")
    elif product == "kiwi":
        print(f"{(2.7 * number_of_products):.2f}")
    elif product == "pineapple":
        print(f"{(5.5 * number_of_products):.2f}")
    elif product == "grapes":
        print(f"{(3.85 * number_of_products):.2f}")
    else:
        print("error")
elif day == "Saturday" or day == "Sunday":
    if product == "banana":
        print(f"{(2.7 * number_of_products):.2f}")
    elif product == "apple":
        print(f"{(1.25 * number_of_products):.2f}")
    elif product == "orange":
        print(f"{(0.9 * number_of_products):.2f}")
    elif product == "grapefruit":
        print(f"{(1.6 * number_of_products):.2f}")
    elif product == "kiwi":
        print(f"{(3 * number_of_products):.2f}")
    elif product == "pineapple":
        print(f"{(5.6 * number_of_products):.2f}")
    elif product == "grapes":
        print(f"{(4.2 * number_of_products):.2f}")
    else:
        print("error")
else:
    print("error")

File: (3) Java/(01) Programming Basics with Java - January 2025/(05) Conditional Statements Advanced - Lab/fruit_shop.java
import java.util.Scanner;

public class fruit_shop {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String fruit = scan.nextLine();
        String dayOfWeek = scan.nextLine();
        double quantity = Double.parseDouble(scan.nextLine());

        double price = 0;

        switch (dayOfWeek) {
            case "Saturday", "Sunday" -> {
                switch (fruit) {
                    case "banana" -> price = 2.7;
                    case "apple" -> price = 1.25;
                    case "orange" -> price = 0.90;
                    case "grapefruit" -> price = 1.6;
                    case "kiwi" -> price = 3;
                    case "pineapple" -> price = 5.6;
                    case "grapes" -> price = 4.2;
                }
            }
            case "Monday", "Tuesday", "Wednesday", "Thursday", "Friday" -> {
                switch (fruit) {
                    case "banana" -> price = 2.5;
                    case "apple" -> price = 1.2;
                    case "orange" -> price = 0.85;
                    case "grapefruit" -> price = 1.45;
                    case "kiwi" -> price = 2.7;
                    case "pineapple" -> price = 5.5;
                    case "grapes" -> price = 3.85;
                }
            }
        }

        double totalPrice = quantity * price;
        if (totalPrice == 0 && quantity != 0) {
            System.out.println("error");
        } else {
            System.out.printf("%.02f", totalPrice);
        }
    }
}
